ui/app/src/components/shared/DetailsPanel.vue

```diff
@@ -61,10 +61,9 @@ export default defineComponent({
           </Tooltip>
         </span>
 
-        <span
-          >{{ formatNumber(minimumReceived) }}
-          <span>{{ toToken.toUpperCase().replace("C", "c") }}</span></span
-        >
+        <span>{{ formatNumber(minimumReceived) }}
+          <span>{{ toToken.slice(1).toUpperCase() }}</span>
```

Comment:
What's this

Reply:
@59023g if we replace C with c it will mess up tokens like USDc

Reply:
but it's `cUSDC`

Reply:
Also, `slice` will remove string character at index 0. Please revert this.

Reply:
Also, slice will remove string character at index 0. Please revert this.